news/2020-11-05/haskell-foundation-announced.markdown

```diff
@@ -0,0 +1,5 @@
+---
+title: Haskell Foundation Announced
+---
+
+Today Simon Peyton-Jones announced the formation of The Haskell Foundation, a non-profit organization focused on increasing adoption of the Haskell programming language. Watch the announcemnt <a href='https://youtu.be/MEmRarBL9kw' target='_blank'>here.</a>
```

Comment:
Does the embedded html thing work? I thought it was the [name](link) syntax?

Reply:
It works on my side (and on other websites), if we want to keep the current behavior : "open in a new tab" with ```target='_blank'``` I don't know another solution than embed html in markdown